src/Template/SentencesLists/index.ctp

```diff
@@ -61,31 +61,190 @@ $this->set('title_for_layout', $this->Pages->formatTitle($title));
     <section class="md-whiteframe-1dp">
         <md-toolbar class="md-hue-2">
             <div class="md-toolbar-tools">
-                <h2><?= $this->safeForAngular($title) ?></h2>
+                <h2 flex>
+                    <?= $this->safeForAngular($title) ?>
+                </h2>
+                <md-menu md-offset="5 50" md-position-mode="target-right target">
+                    <md-button ng-click="$mdOpenMenu($event)">
+                        <md-icon>sort</md-icon> Sort by
+                    </md-button>
+                    <md-menu-content>
+                        <!-- <md-menu-item ng-repeat="item in ['list', 2, 22223, 9324, 02934, 12]">
+                        <md-button>
+                        <md-icon>{{ $index === 1 ? 'check' : 'blank'}}</md-icon>
+                        <span style="padding-right: 12px">Option {{item}}</span>
+                        </md-button>
+                        </md-menu-item> -->
+                        
+                        <md-menu-item>
+                            <md-button>
+                                <md-icon>
+                                    <?php 
+                                        if (strcmp($this->Paginator->sortKey(),'name') == 0 && strcmp($this->Paginator->sortDir(),'asc') == 0) {
+                                            echo 'check'; 
+                                        }
+                                        else {
+                                            echo 'blank';
+                                        }
+                                    ?>
+                                </md-icon>
+                                <span style="padding-right: 12px">
+                                    <?php echo $this->Paginator->sort('name', __('list name'), ['direction'=>'asc', 'lock'=>true]); ?>
+                                </span>
+                            </md-button>
+                        </md-menu-item>
```

Comment:
You could consider creating an [element](https://book.cakephp.org/2/en/views.html#elements) for these `<md-menu-item>`.

Then here you would just have something like this:
```
echo $this->element('sort_option', [
    'param' => 'name',
    'direction' => 'asc',
    'label' => __('list name')
]);
````

Reply:
Yes I thought my solution was not the most elegant. 
Now I made two of these Elements as the first page I made has a couple of lines with different code, but since it is used inly twice, I could change it and add a parameter to make only one.